modules/search/instant-search/components/overlay.scss

```diff
@@ -1,4 +1,4 @@
-$overlay-vertical-padding: 3.5em;
+$overlay-vertical-padding: 4em;
```

Comment:
This is to give enough space for the outline styles without overlapping the search box.

<img width="1089" alt="Screen Shot 2020-09-29 at 16 40 18" src="https://user-images.githubusercontent.com/17325/94510131-7c4ed300-0272-11eb-8f16-369d33d24111.png">